### DISCUSSION TITLE
custom scenario to select 2 of 9 different products per time slot

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3086] [Q&A]
Hello

I have a case that I would like to create 9 different products for an event series with a quota of 25 per product. Also I have two time slots that all of these products are available.
I would like
1. to allow the customer to select up to 2 of these 9 products for each time slot.
2. When someone books one product, I want the product to become unavailable whether its quota has been reached or not.

Thank u in advance

**Comment by raphaelm** [comment]
These might only be possible if you create just one product with a quota of 1 for each time slot, and handle everything else as add-on products
